src/main/java/uk/gov/justice/digital/nomis/service/UsersService.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.digital.nomis.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.justice.digital.nomis.api.UserDetail;
+import uk.gov.justice.digital.nomis.jpa.repository.StaffUserAccountRepository;
+import uk.gov.justice.digital.nomis.service.transformer.UserTransformer;
+
+import java.util.Optional;
+
+@Service
+public class UsersService {
+
+    private final UserTransformer userTransformer;
+    private final StaffUserAccountRepository staffUserAccountRepository;
```

Comment:
Where is this StaffUserAccountRepository? Is it in this review?

Reply:
It was in a previous review you did